action.yaml

```diff
@@ -40,6 +40,9 @@ inputs:
     description: Which action flyway will perform (ie. migrate or repair)
     required: true
     default: migrate
+  environment:
+    description: The Openr environment (dev, tech, prod)
```

Comment:
Can we use stag or staging instead of tech? Staging is consistent throughout all other repos

Reply:
Maybe default to dev?

Reply:
Also though, see corresponding comment: https://github.com/weareopenr/api/pull/1828/files#r1617015683

Reply:
@Smittey , this is how we are resolving it: https://github.com/weareopenr/api/pull/1828/files#:~:text=CASE%20%27%24%7Benvironment%7D%27

I just made sure this isn't a required arg too